packages/mattermost-redux/src/reducers/entities/threads/counts.ts

```diff
@@ -93,6 +95,23 @@ function handleLeaveChannel(state: ThreadsState['counts'] = {}, action: GenericA
     };
 }
 
+function handleDecrementThreadCounts(state: ThreadsState['counts'], action: GenericAction) {
+    const {teamId, replies, mentions} = action;
+    const counts = state[teamId];
+    if (!counts) {
+        return state;
+    }
+
+    return {
+        ...state,
+        [teamId]: {
+            total: Math.max(counts.total - 1, 0),
+            total_unread_mentions: Math.max(counts.total_unread_mentions - mentions, 0) || 0,
```

Comment:
you are right there is no need to!

Reply:
well, if you can make sure that `counts.total_unread_mentions - mentions` will always be a number (or at least a stringified number), otherwise `Math.max` will return `NaN`

Reply:
that was the reason for `|| 0` but it should always be defined at that point